.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +23,7 @@ Fixes #<issue_number>
 - [ ] Did you write any new necessary tests? (not for typos and docs)
 - [ ] Did you verify new and existing tests pass locally with your changes?
 - [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)
+- [ ] Did you list all the breaking changes introduce by this pull request ?
```

Comment:
is this needed while there are people who do not border with filling the checklist at all?

Reply:
Yes, I believe it is needed.